gpep517/scheme.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-FileCopyrightText:
+# SPDX-License-Identifier: MIT
+
```

Comment:
While these headers are all fun, they're inconsistent with the rest of the codebase.

Reply:
I see I forgot to update the first line to include the pypa/installer copyright line.

I don't care what format the headers are in, and I'm not trying to be "fun". I do, however, think that it is important when you are copying code from another project to state where you got it, who wrote it, and what it's licensed under. That is the purpose of having a separate copyright declaration that is not part of "the rest of the codebase".